server/models/Users.js

```diff
@@ -24,6 +25,53 @@ class Users {
       },
     }));
   }
+
+  static userLogin(req, res) {
+    const connectString = database;
+    const clientString = new Client(connectString);
+    clientString.connect();
+
+    // Check if user exist
+    const results = [];
+    const checkExistingUser = clientString.query(`SELECT * FROM users WHERE email= '${req.body.email}'`);
+    checkExistingUser.on('row', (row) => {
+      results.push(row);
+    });
+
+    // After all data is returned, close connection and return results
+    checkExistingUser.on('end', () => {
+      if (results.length <= 0) {
+        return res.status(403).json({
```

Comment:
is there a reason why you are returning a status code for forbidden here? Please check for the appropriate status code.